app/views/granules/prepare_script.html.erb

```diff
@@ -43,7 +43,10 @@
 <body>
 <div id="errors"></div>
 <div id="loading">Preparing download script. Fetching downloadable granule links...</div>
-<div id="download-description" style="display: none;">Script preparation completed. Please click the button to download the script. <a href="#" id="download-button">Download Script File</a></div>
+<div id="download-description" style="display: none;">
+  <p>Script preparation completed. Please click the button to download the script. <a href="#" id="download-button">Download Script File</a></p>
+  <p>For a detailed walk through of this process, please reference this <a target="_blank" href='https://wiki.earthdata.nasa.gov/display/EDSC/How+To%3A+Use+the+Download+Access+Script'>How To guide</a>.</p>
```

Comment:
@jboukes, I don't think I was very clear in my first comment. I thought that this line of text should be added in addition to the text you had before. I would add this after the Linux instructions, before the Windows instructions.

Reply:
Woops!  Yes, that's definitely different - I'll update the PR in 5, 4, 3...